tests/operator/operator.go

```diff
@@ -1350,21 +1350,13 @@ spec:
 					return err
 				}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 
-				// change run stategy to halted to be able to restore the vm
-				Eventually(func() bool {
-					vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(nil)).Get(context.Background(), vmYaml.vmName, metav1.GetOptions{})
-					if err != nil {
-						return false
-					}
-					vm.Spec.RunStrategy = &runStrategyHalted
-					_, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Update(context.Background(), vm, metav1.UpdateOptions{})
-					if err != nil {
-						return false
-					}
-					updatedVM, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Get(context.Background(), vmYaml.vmName, metav1.GetOptions{})
-					Expect(err).ToNot(HaveOccurred())
-					return updatedVM.Spec.Running == nil && updatedVM.Spec.RunStrategy != nil && *updatedVM.Spec.RunStrategy == runStrategyHalted
-				}, 30*time.Second, 3*time.Second).Should(BeTrue())
+				By("Changing run stategy to halted to be able to restore the vm")
+				patches, err := patch.New(patch.WithAdd("/spec/runStrategy", &runStrategyHalted)).GeneratePayload()
+				vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(nil)).Patch(context.Background(), vmYaml.vmName, types.JSONPatchType, patches, metav1.PatchOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(vm.Spec.Running).ToNot(BeNil())
```

Comment:
Why this assert is required, if we can check only `vm.Spec.RunStrategy`?

Reply:
I think you're right. fixed.

Reply:
@RamLavi I think you didn't push the changes